vmware/ngc/NGC-AdapterManager/src/main/resources/constants.properties

```diff
@@ -2,4 +2,5 @@ OPENSDS_TENANT=admin
 OPENSDS_DOMAIN=default
 OPENSDS_AVAILABILITYZONE=default
 OPENSDS_STORAGENAME=OpenSDS-LVM
-OPENSDS_VENDOR=OpenSDS
\ No newline at end of file
+OPENSDS_VENDOR=OpenSDS
+OPENSDS_HOST_ACCESSMODE=agentless
```

Comment:
Can we say it as OPENSDS_HOST_ACCESSMODE_AGENTLESS? In future we can have Agent mode too

Reply:
We need to assign the value to this constant

Reply:
What my point was that if we support agent mode.. this constant name is too generic